pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -79,6 +81,7 @@ var defaultGatheringRules = []GatheringRule{
 			},
 		},
 	},
+	// GatherAPIRequestCounts
```

Comment:
BTW, I added these comments to help to identify the conditions for each gather. It was a little bit hard for me to read it.

Reply:
Makes sense and I agree that these lists tend to be difficult to read, especially as they grow longer.